script/generate-config-gypi.py

```diff
@@ -57,6 +57,8 @@ def main(target_file, target_cpu):
   v['node_module_version'] = int(args['node_module_version'])
   # Used by certain versions of node-gyp.
   v['build_v8_with_gn'] = 'false'
+  # Enable whole-program optimization for electron native modules.
+  v['node-with-ltcg'] = 'true'
```

Comment:
This should be done by adding the arg in `run_node_configure`, since the variable name might be changed by upstream.

```js
args += ['--with-ltcg']
```